### ISSUE TITLE
Failed loading [...]: [...] matched no files or resources on Windows

### ISSUE THREAD

**Issue by sveinatle** [busterjs/buster#165]
buster-test fails to load source or test files specified in the buster.js config file on Windows. 
#### To reproduce

1 - Create folder      `C:\MyProject\test`
2 - Create file        `C:\MyProject\test\MyModule.Test.js`
3 - Create config file `C:\MyProject\buster.js`:

``` javascript
var config = module.exports;

config["My tests"] = {
    env: "node",
    tests: [
        "test/*.Test.js"
    ]
}
```

 4 - Open command line at `C:\MyProject\` and run `buster-test`.
#### Result

```
Failed loading test/*.Test.js: 'C:\MyProject\test\MyModule.Test.js' matched no files
```
#### Additional information

I have tried various combinations of:
- putting buster.js in the `test/` folder and setting rootPath to `../`,
- using `\` or `\\` instead of `/`, and
- naming MyModule.Test.js explicitly in the config instead of using wildcards,

but none work. Notice that even when using wildcards, buster-test still finds the correct file at some point, but it fails a match later on somewhere else. When using `\` the slash escapes the next character (output from buster-test shows the next character has been eaten), so that's obviously a no-go. Using `\\` makes buster-test output the correct string, but it doesn't actually match the file at all (unlike `/`) -- compare:

```
C:\MyProject>buster-test
Failed loading test\MyModule.Test.js: test\MyModule.Test.js matched no files or resources

C:\MyProject>buster-test
Failed loading test/MyModule.Test.js: 'C:\MyProject\test\MyModule.Test.js' matched no files
```

So to summarize, `/` and `\\` actually cause two different errors.
